tests/test_model.py

```diff
@@ -113,6 +113,108 @@ def test_call_deprecated(model, output_names):
             )
 
 
+@pytest.mark.parametrize(
+    'model, outputs, expected',
+    [
+        (
+            audonnx.Model(audonnx.testing.create_model_proto([[1, -1]])),
+            None,
+            pytest.SIGNAL,
+        ),
+        (
+            audonnx.testing.create_model([[-1], [-1]]),
+            None,
+            np.zeros([2, pytest.SIGNAL.shape[1]], dtype=np.float32),
```

Comment:
Shouldn't the returned signal be of one dimension with the length of two times `pytest.SIGNAL`?